src/app-layout/index.tsx

```diff
@@ -484,25 +488,19 @@ const OldAppLayout = React.forwardRef(
                 }}
               >
                 {notifications && (
-                  <DarkHeader
```

Comment:
DarkHeader seems to be used to add position sticky https://github.com/cloudscape-design/components/blob/main/src/app-layout/dark-header/index.tsx. If we apply it to <Notifications>, no need of DarkHeader anymore?

Reply:
There is no need in DarkHeader in this file at all. Fine for me to remove (bonus points the remaining ones and get rid of the import in this file)